tests/forms_tests/field_tests/filepathfield_test_dir/README

```diff
@@ -0,0 +1,3 @@
+********************************************************************************************
```

Comment:
I'm not sure this is necessary. Obviously tests will fail if it's modified and there might be a need to modify it for future tests.